.eslintrc.yml

```diff
@@ -9,7 +9,7 @@ extends:
   - airbnb-typescript
   - plugin:jest/recommended
   - plugin:jest/style
-  - plugin:prettier/recommended
```

Comment:
Does this mean we can uninstall `eslint-plugin-prettier`? Or is it still needed?

Reply:
ah yes! doing this now.